README.md

```diff
@@ -212,67 +215,65 @@ For example:
 
-See [Fabric questions on Stackoverflow](http://stackoverflow.com/questions/tagged/fabricjs),
-Fabric snippets on [jsfiddle](http://jsfiddle.net/user/fabricjs/fiddles/)
-or [codepen.io](http://codepen.io/tag/fabricjs).
+## More resources
 
-Fabric on [LibKnot](http://libknot.ohmztech.com/).
+- [Fabric.js on Twitter](https://twitter.com/fabricjs)
+- [Fabric.js on CodeTriage](https://www.codetriage.com/kangax/fabric.js)
+- [Fabric.js on Stackoverflow](https://stackoverflow.com/questions/tagged/fabricjs)
+- [Fabric.js on jsfiddle](https://jsfiddle.net/user/fabricjs/fiddles/)
+- [Fabric.js on Codepen.io](https://codepen.io/tag/fabricjs)
+- [Presentation from BK.js](http://www.slideshare.net/kangax/fabricjs-building-acanvaslibrarybk)
+- [Presentation from Falsy Values](http://www.slideshare.net/kangax/fabric-falsy-values-8067834)
 
-Get help in Fabric's IRC channel — irc://irc.freenode.net/#fabric.js
 
-### Credits
+## Credits
 
+- [@kangax](https://twitter.com/kangax)
 - [Andrea Bogazzi](https://twitter.com/AndreaBogazzi) for help with bugs, new features, documentation, GitHub issues
 - Ernest Delgado for the original idea of [manipulating images on canvas](http://www.ernestdelgado.com/archive/canvas/)
 - [Maxim "hakunin" Chernyak](http://twitter.com/hakunin) for ideas, and help with various parts of the library throughout its life
 - [Sergey Nisnevich](http://nisnya.com) for help with geometry logic
 - [Stefan Kienzle](https://twitter.com/kienzle_s) for help with bugs, features, documentation, GitHub issues
-- [Shutterstock](http://www.shutterstock.com/jobs) for the time and resources invested in using and improving fabric.js
+- [Shutterstock](http://www.shutterstock.com/jobs) for the time and resources invested in using and improving Fabric.js
 - [And all the other GitHub contributors](https://github.com/kangax/fabric.js/graphs/contributors)
 
-### MIT License
+
+## Sponsor authors
+
+- https://flattr.com/@kangax
+- https://github.com/sponsors/asturur
+- https://www.patreon.com/fabricJS
+
+## History
+
+Fabric.js started as a foundation for design editor on [printio.ru](http://printio.ru) — interactive online store with ability to create your own designs. The idea was to create [Javascript-based editor](http://printio.ru/ringer_man_tees/new), which would make it easy to manipulate vector shapes and images on T-Shirts. Since performance was one of the most critical requirements, we chose canvas over SVG. While SVG is excellent with static shapes, it's not as performant as canvas when it comes to dynamic manipulation of objects (movement, scaling, rotation, etc.). Fabric.js was heavily inspired by [Ernest Delgado's canvas experiment](http://www.ernestdelgado.com/public-tests/canvasphoto/demo/canvas.html). In fact, code from Ernest's experiment was the foundation of an entire framework. Later, Fabric.js grew into a collection of distinct object types and got an SVG-to-canvas parser.
```

Comment:
```suggestion
Fabric.js started as a foundation for design editor on [printio.ru](http://printio.ru) — interactive online store with ability to create your own designs. The idea was to create [Javascript-based editor](http://printio.ru/ringer_man_tees/new), which would make it easy to manipulate vector shapes and images on T-Shirts. Since performance was one of the most critical requirements, we chose canvas over SVG. While SVG is excellent with static shapes, it's not as performant as canvas when it comes to dynamic manipulation of objects (movement, scaling, rotation, etc.). Fabric.js was heavily inspired by a canvas experiment created by Ernest Delgado. In fact, code from Ernest's experiment was the foundation of an entire framework. Later, Fabric.js grew into a collection of distinct object types and got an SVG-to-canvas parser.
```

This canvas experiment URL is actually long dead and I can find no new location for it so probably best to just remove the link.

Reply:
actually nevermind, @asturur is right this whole section is kind of unnecessary

> Yes History can be removed, we should just credit that demo developer in the credits.